Documentation/Architecture/ConfigurationAndProfiles.md

```diff
@@ -0,0 +1,85 @@
+# Configuration and profiles
+
+Not every single consumer of the MRTK will want it to behave the same way - some will want to have the spatial
+mesh running when on AR devices that support it. Some may want the diagnostic visualization on all the time,
+and some may only want it on when the user says a voice command. 
+
+The MRTK needs to be configurable in order to support a wide range of those requirements, and it uses a concept
+called 'profiles' to accomplish this.
+
+## What is a profile?
+
+A profile is a serialized asset (an instance of a ScriptableObject) which contains properties that configure
+the initialization and runtime behavior of an MRTK service. For example, the MRTK has a camera service, which
```

Comment:
>For example, the MRTK has a camera service, which will apply different properties to the main camera, depending on whether or not the display is transparent (like in the case of a HoloLens) or opaque (like in the case of a VR headset). The camera service is given a [camera profile](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit/Definitions/MixedRealityCameraProfile.cs),
which contains those different transparent vs. opaque settings.

How about:

_For example, a [Camera System Profile](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit/Definitions/MixedRealityCameraProfile.cs) stores your main camera's settings for both VR and for HoloLens apps. When the MixedRealityCameraSystem is initialized, it's given a reference to your profile. While running it detects your platform and copies the appropriate settings from your profile to your main camera._

_[Screenshot of camera profile settings] [Screenshot of main camera's settings]_